### PR TITLE
Update Makefile for Netlify deployments

### PR DESCRIPTION
**Pull request by danielledeleo**
The Minify and gzip steps are no longer needed because Netlify handles it transparently. SECURITY.txt is now fetched from the unit repository during deployment.

### CODE REVIEW

File: security.txt
Comment (ac000) [inline security.txt:1]: I would probably split the removal of this file out into its own commit as it's not _directly_ related to the Makefile changes.

File: Makefile [hunk 1]
```diff
@@ -20,15 +18,6 @@ EXCLUDE = \
...
-COMPRESS = -size +1000c \
-	\( -name '*.html' \
-	-o -name '*.css' \
-	-o -name '*.js' \
-	-o -name '*.svg' \
-	-o -name '*.txt' \
-	-o -name '*.xml' \)
-
-
 .PHONY: site serve check clean deploy do_gzip
```
Comment (ac000) [inline Makefile:21]: I think do_gziip can be removed from here now.

Perhaps ping could be added though... looks like it got overlooked at some point...

**Review by ac000 — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-GOOGLE		?= https://www.google.com/webmasters/tools/ping?sitemap=
-BING		?= http://www.bing.com/ping?sitemap=
-
-# https://github.com/tdewolff/minify/tree/master/cmd/minify
-MINIFY		?= minify
+SITEMAP			?= python3 sitemaps.py
+URL				?= https://unit.nginx.org
+GOOGLE			?= https://www.google.com/webmasters/tools/ping?sitemap=
+BING			?= http://www.bing.com/ping?sitemap=
+UNIT_SECURITY	?= https://github.com/nginx/unit/raw/master/SECURITY.txt
```
Comment (callahad) [inline Makefile:8]: Looks like your editor and the previous revision of the file had different widths set for tabstops, causing misalignment in some cases, including when viewing this diff on GitHub.

Probably worth either re-aligning with tabs set to be 8 chars wide to match prior practice?

**Comment by ac000** [comment]
You _could_ mention the ping target is also no longer required. Other than that, looks good.

```
Reviewed-by: Andrew Clayton <a.clayton@nginx.com>
```

**Review by ac000 — APPROVED** [review]

**Pull request merged**